dbaas/api/task.py

```diff
@@ -6,9 +6,19 @@
 from logical.models import Database, DatabaseHistory
 
 
+RELEVANCE_DICT = {
```

Comment:
Could you please remove this code, I think it's duplicated from:
https://github.com/globocom/database-as-a-service/pull/391/files#diff-6e1eeba35c8541a04ee08b857f42cfafR38